server/testing/src/main/java/org/apache/james/util/docker/Images.java

```diff
@@ -20,7 +20,7 @@
 package org.apache.james.util.docker;
 
 public interface Images {
-    String FAKE_SMTP = "weave/rest-smtp-sink:latest";
+    String FAKE_SMTP = "quanth99/rest-smtp-sink:1.0";
```

Comment:
If you can comment the URL to the Dockerfile project on github then that's perfect!

Reply:
done